### ISSUE TITLE
Create Character Graphics

### ISSUE THREAD

**Issue by Master-Foo** [Master-Foo/game#2]
The images used for character animations are stolen.

Somebody make some custom animations.

No programming is necessary. Just replace the graphics in /resources/images/...

**Comment by jrslagle** [comment]
I really like the idea of doing the 3D sketch up and putting a set of renders in the game. Makes the transition to 3D that much easier.

**Comment by Master-Foo** [comment]
Yeah, I was thinking, we can start working on 3D assets now for use in the future. But in the mean time just use "Canned 3D" images as 2D sprites.

The question is, who wants to do it? Anyone who wants to tackle this task, assign themselves to this issue in the right side bar of this page.

**Comment by jrslagle** [comment]
I'm not claiming this unless I decide to follow through. But on the topic of assigning yourself, I see Assignees on the right, but nothing over there is active. Do we have permission?

**Comment by Master-Foo** [comment]
I don't know what you can and can't do. All I know is that I can do it. Find a issue you want to own and try to assign yourself to it. If you can't tell me. I added y'all as colaborators to this project. I'm assuming privileges come with that.

**Comment by Master-Foo** [comment]
BTW, I think you need to check your email to verify your collaborator status. Make sure you are actually a collaborator.

**Comment by Master-Foo** [comment]
According to the "Add Collaborators" page, you haven't confirmed yet.

**Comment by jrslagle** [comment]
@techwulf and I have been playing hot potato with this one. This is my priority after #25 if he doesn't get to it first.

**Comment by Master-Foo** [comment]
@jrslagle you still need to clean up that pull request you submitted days ago. Do that before anything else.

**Comment by jrslagle** [comment]
@Master-Foo 

K. I've got one person working on concept art, and another up for modeling him/her. 

Were you thinking something more silly, realistic, futuristic? The head to body ratio says a lot. I've been imagining a character more like a chibi toon, like a kid with a huge head. I'm up for a lot of art styles - Borderlands, Diablo 3, Final Fantasy 7 cinematics, FF9. But now's the time to talk about style since we're creating concept art.

**Comment by jrslagle** [comment]
https://goo.gl/photos/3SogYXT252LaLBFH6

**Comment by jrslagle** [comment]
I asked if she could make the helmet look more like a motorcycle helmet. But I'm generally on board with the wet suit and armor idea.

**Comment by jrslagle** [comment]
Here's an example of her style. 

https://goo.gl/photos/P3KChCSP662xnWbb6

**Comment by Master-Foo** [comment]
If someone wants to do the art, I'm pretty generous with letting them do what they want to do.

The famous "Joshua Maxim" is "The person who gets shit done decides how shit gets done." Serious, the "Joshua Maxim" is a real thing, it's a term Blake invented. Verify with @techwulf.

I suppose if I were to have any wishes on how it gets done. It's to "Keep it Simple". That much should be pretty obvious, I've been saying this from the beginning. I'd rather this person spend their time making lots of simple resources we can use than one complicated resource.

As for style, I'm not sold on any given gimmick. I'll leave that up to the inspiration of whoever is doing the work. I suppose that if this is going to be a "Hard Sci-Fi Genera" the style should probably reflect that. However, given the resources we have, it's more realistic to do something like toon-shading with low-poly models than it is to try and make a super-realistic design.

Fuck it, tell this person to make a model of a 3D spherical cow, we'll just use that. Welcome to the team.

**Comment by jrslagle** [comment]
I got about 40% through this on Dec-10. And if anyone wants to take up the torch, @techwulf. I recommend replicating this tutorial. It'll give us animations of a naked person running around. It shows you how to put motion capture data onto make human models in blender. All free plug and play.

Easy Walk Animation with Blender and Makehuman
https://www.youtube.com/watch?v=2hylWVut7z0

His link to the mocap data is outdated, but I found this nearby.
https://accad.osu.edu/research/mocap/mocap_data.htm
